documentation/dev_running_from_source.markdown

```diff
@@ -159,6 +159,25 @@ run the following:
 
     $ lein run -c /path/to/puppetserver.conf
 
+Running the Agent
+-----
+
+Use a command like the one below to run an agent against your running puppetserver:
+
+        puppet agent --confdir ~/.puppetlabs/etc/puppet \
```

Comment:
On Debian, not sure where else, it's not looking there, it wants to find it in ~/.puppet for whatever reason. I'm happy to put caveats or something else above, but I know I've fumbled around trying to remember the right config options to make this work and they aren't documented.